src/lasuite/marketing/backends/brevo.py

```diff
@@ -0,0 +1,85 @@
+"""Brevo marketing automation integration."""
+
+import logging
+
+import brevo_python
+import urllib3
+
+from lasuite.marketing.backends import ContactData
+from lasuite.marketing.exceptions import ContactCreationError
+
+from .base import BaseBackend
+
+logger = logging.getLogger(__name__)
+
+
+class BrevoBackend(BaseBackend):
+    """
+    Brevo marketing automation integration.
+
+    Handles:
+    - Contact management and segmentation
+    - Marketing campaigns and automation
+    - Email communications
+    """
+
+    def __init__(self, api_key: str, api_contact_list_ids: list[int], api_contact_attributes: dict | None = None):
+        """Configure the Brevo backend."""
+        configuration = brevo_python.Configuration()
+        configuration.api_key["api-key"] = api_key
+
+        self._api_client = brevo_python.ApiClient(configuration)
+        self.api_contact_attributes = api_contact_attributes or {}
+        self.api_contact_list_ids = api_contact_list_ids
+
+    def create_or_update_contact(self, contact_data: ContactData, timeout: int = None) -> dict:
+        """
+        Create or update a Brevo contact.
+
+        Args:
+            contact_data: Contact information and attributes
+            timeout: API request timeout in seconds
+
+        Returns:
+            dict: Brevo API response
+
+        Raises:
+            ContactCreationError: If contact creation fails
+            ImproperlyConfigured: If required settings are missing
+
+        Note:
+            Contact attributes must be pre-configured in Brevo.
+            Changes to attributes can impact existing workflows.
+
+        """
+        contact_api = brevo_python.ContactsApi(self._api_client)
+
+        attributes = {
+            **self.api_contact_attributes,
+            **(contact_data.attributes or {}),
+        }
+
+        list_ids = (contact_data.list_ids or []) + self.api_contact_list_ids
+
+        contact = brevo_python.CreateContact(
+            email=contact_data.email,
+            attributes=attributes,
+            list_ids=list_ids,
```

Comment:
I'm afraid a contact in a product removes lists from other product if the contact already exists